### PR TITLE
docs: add community-expectations.md doc

### LINKED ISSUE #159: repo: add additional community docs to the openstack-helm repos
**Kubernetes Version** (output of `kubectl version`): N/A

**Helm Client and Tiller Versions** (output of `helm version`): N/A

**Development or Deployment Environment?**: N/A

**Release Tag or Master**: N/A

**Expected Behavior**: N/A

**What Actually Happened**: N/A

**How to Reproduce the Issue** (as minimally as possible): N/A

**Any Additional Comments**: 
As we prepare our repo for the community, we need to address and include the following in our repository:

- changelog
- code of conduct
- contributing guidelines
- contributor agreements (?)
- expectations (for issues, pr's, contributors and reviewers)
- OWNERS
- RELEASES

### PR DESCRIPTION
**Pull request by wilkers-steve**
Adds community-expectations.md, outlining expected behavior of
OpenStack-Helm contributors

**What is the purpose of this pull request?**:  Add community expectations doc

**What issue does this pull request address?**: Fixes #159 

**Notes for reviewers to consider**:

**Specific reviewers for pull request**: @v1k0d3n

### CODE REVIEW

File: docs/community-expectations.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+We expect all community members to adhere to the OpenStack Code of Conduct found [here](https://www.openstack.org/legal/community-code-of-conduct/).
+
+### Code Review
+
+The OpenStack-Helm community believes in the value quality code reviews bring to the project.  Thorough code reviews increase the quality and readability of the project's codebase, and ultimately makes OpenStack-Helm a more robust platform.  However, code reviews introduce additional work for all parties involved.  In order to reduce the overhead quality code reviews introduce, we expect all active community members to be active reviewers.
+
+#### Expectations of Reviewers: Review Comments
+
+As reviewers are often the first contact new contributors have with the project, it's important that active reviewers act with respect and thoughtfulness when rreviewing pull requests.  Review comments should never be personal or condescending.
```
Comment (gardlt) [inline docs/community-expectations.md]: excess r in rreviewing.

File: docs/community-expectations.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Community Expectations
+
+OpenStack-Helm is a community project.  The primary goal of the OpenStack-Helm project is to provide a community-driven solution for the challenges of orchestrating and managing an OpenStack control plane with Kubernetes.  As a result, the project depends on the members of the community to provide a friendly and collaborative environment in order to meet that goal.
+
+To achieve our goals, we've listed expectations for the OpenStack-Helm community below.  This living document shall evolve with the project through the same pull request and code review process that shapes the rest of the project.  It covers the expectations of conduct that govern all members of the community as well as the expectations around code review that govern all active contributors to Kubernetes.
+
+### Code of Conduct
+
+We expect all community members to adhere to the OpenStack Code of Conduct found [here](https://www.openstack.org/legal/community-code-of-conduct/).
```
Comment (gardlt) [inline docs/community-expectations.md]: would we like to include the CNCF code of conduct link as well?

Comment (wilkers-steve) [inline docs/community-expectations.md]: I think we should keep the openstack code of conduct for now.  If necessary, we can add the kubernetes code of conduct at a later time. A lot of this really depends on where the project ends up in the future, so the openstack code of conduct is a safe default for now

File: docs/community-expectations.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+As reviewers are often the first contact new contributors have with the project, it's important that active reviewers act with respect and thoughtfulness when reviewing pull requests.  Review comments should never be personal or condescending.
+
+#### Expectations of Reviewers: Review Latency
+
+Reviewers should respond to pull requests in a timely fashion.  By reducing review latency, we can ensure the project goals are met in a timely fashion.  Furthermore, we ensure new contributors feel their pull requests are valuable.
+
+## Thanks
+
+We'd like to thank everyone who contributes their time and effor to making OpenStack-Helm both a successful project and an extraordinary community!
```
Comment (larryrensing) [inline docs/community-expectations.md]: effort

**Comment by wilkers-steve** [comment]
@larryrensing fixed

File: docs/community-expectations.md (hunk 2 above)
Comment (v1k0d3n) [inline docs/community-expectations.md]: +1 on Openstack Code of Conduct.

File: docs/community-expectations.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+[![Travis CI](https://travis-ci.org/att-comdev/openstack-helm.svg?branch=master)](https://travis-ci.org/att-comdev/openstack-helm)
+
+## Community Expectations
+
+OpenStack-Helm is a community project.  The primary goal of the OpenStack-Helm project is to provide a community-driven solution for the challenges of orchestrating and managing an OpenStack control plane with Kubernetes.  As a result, the project depends on the members of the community to provide a friendly and collaborative environment in order to meet that goal.
+
+To achieve our goals, we've listed expectations for the OpenStack-Helm community below.  This living document shall evolve with the project through the same pull request and code review process that shapes the rest of the project.  It covers the expectations of conduct that govern all members of the community as well as the expectations around code review that govern all active contributors to Kubernetes.
```
Comment (v1k0d3n) [inline docs/community-expectations.md]: "To achieve our goals"

to 

"To achieve these goals"

**Comment by wilkers-steve** [comment]
@v1k0d3n made some changes here.  If we still want this, we can add it in.

**Comment by v1k0d3n** [comment]
do we still need or want this @wilkers-steve, or do we want to wait until we're actually in openstack? (i'm leaning towards waiting). but that would mean closing this PR to reduce our carry-over/loose ends. thoughts?

**Comment by wilkers-steve** [comment]
I'm okay with closing this out.  We can revisit this once we move

**Comment by v1k0d3n** [comment]
sounds good.

**Pull request closed without merging**